src/Shared/E2ETesting/BrowserFixture.cs

```diff
@@ -143,5 +154,54 @@ private async Task<(IWebDriver browser, ILogs log)> CreateBrowserAsync(string co
 
             throw new InvalidOperationException("Couldn't create a Selenium remote driver client. The server is irresponsive");
         }
+
+        private async Task<(IWebDriver browser, ILogs log)> CreateSauceBrowserAsync(string context, ITestOutputHelper output)
+        {
+            var name = "Blazor E2E tests";
+            if (!string.IsNullOrEmpty(context))
+            {
+                name = $"{name} - {context}";
+            }
+
+            var capabilities = new DesiredCapabilities();
+            capabilities.SetCapability("username", E2ETestOptions.Instance.Sauce.Username);
+            capabilities.SetCapability("accessKey", E2ETestOptions.Instance.Sauce.AccessKey);
+            capabilities.SetCapability("tunnelIdentifier", E2ETestOptions.Instance.Sauce.TunnelIdentifier);
+            capabilities.SetCapability("name", name);
+
+            capabilities.SetCapability("browserName", "Chrome");
```

Comment:
I know this is currently hardcoded. But this will come from config or env variables.

Reply:
This is fine for now, although you could simply add these to the config file that E2E tests use